st/clitests/framework.py

```diff
@@ -161,6 +161,22 @@ def command_response_should_have(self, msg):
         print("Response has [%s]." % (msg))
         return self
 
+    def command_response_should_have_n_times(self, msg, count):
+        temp_count = 0
+        if not Config.dummy_run:
+            #counting the number of occurences of msg("should be equal to 'count' times")
+            text = self.status.stdout
+            index = 0
+            while index < len(text):
+                index = text.find(msg, index)
+                if index == -1:
+                    break
+                index += 8
```

Comment:
dont hard code it, you need to get length of the filename passed

Reply:
done